gpiozero/spi_devices.py

```diff
@@ -128,10 +167,9 @@ def channel(self):
     @property
     def differential(self):
         """
-        If ``True``, the device is operated in pseudo-differential mode. In
-        this mode one channel (specified by the channel attribute) is read
-        relative to the value of a second channel (implied by the chip's
-        design).
+        If ``True``, the device is operated in differential mode. In this mode
+        one channel (specified by the channel attribute) is read relative to
+        the value of a second channel (implied by the chip's design).
```

Comment:
Damn ... that's probably a copy'n'paste mistake. It is meant to read pseudo- on the MCP3xxx devices and not on the MCP33xx ones.